config/prod.exs

```diff
@@ -1,9 +1,17 @@
 use Mix.Config
 
-config :logger, level: :error
-
 config :mozart_fetcher, timeout: 3000
 
 config :ex_metrics,
   send_metrics: true,
   raise_on_undefined_metrics: false
+
+config :logger,
+  backends: [{LoggerFileBackend, :file}]
+
+config :logger, :file,
+  path: "/var/log/mozart_fetcher/error.log",
```

Comment:
Happy with this being hard coded for now, but a future improvement will be to get this from cosmos config/environment variables

Reply:
How do we decide what should be in these "application" configs set at compile time - and what should be put in cosmos? I see the benefit of cosmos applications being that they can change config values between environments - but this log file path won't change between environments, and could be kept here in the `prod.exs` config.

Reply:
Hmm, good point. I was thinking more from the point of view that the log path is used by more than just the application. We might need to mention the logpath in the systemd unit file, and the config for the td-agent

Reply:
ah yes, good point